s3/src/main/java/com/instaclustr/kafka/connect/s3/sink/TopicPartitionBuffer.java

```diff
@@ -98,5 +94,5 @@ public long getEndOffset() {
     public TopicPartition toTopicPartition() {
         return new TopicPartition(topic, partition);
     }
-
 }
+
```

Comment:
In general, I don't think it's necessary to change whitespace in an existing file you otherwise haven't touched.

Reply:
sure i will take care.